apps/omg_watcher/lib/omg_watcher/exit_processor/exit_info.ex

```diff
@@ -80,20 +99,23 @@ defmodule OMG.Watcher.ExitProcessor.ExitInfo do
            exit_id: exit_id,
            exiting_txbytes: exiting_txbytes,
            is_active: is_active,
-           eth_height: eth_height
+           eth_height: eth_height,
+           root_chain_txhash: root_chain_txhash
```

Comment:
same as above.
`event. root_chain_txhash`
you can also remove these guards, I don't know who are they protecting us from.

Reply:
Done 6287e2b.